packages/gasket-plugin-lifecycle/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@gasket/plugin-lifecycle",
   "version": "6.7.3",
   "description": "Allows a gasket/ directory to be used for lifecycle hooks in applications.",
-  "main": "lib/index.js",
```

Comment:
Maybe leave the `main` property, but correct the value, as we have in other plugins:
```
  "main": "index.js"
```

Though this is the default according to [the docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main).

Reply:
Correct, the default is `index.js`, which was why I removed the property entirely, but happy to add it back if it adds clarity.